vote/src/vote_account.rs

```diff
@@ -68,22 +68,49 @@ impl VoteAccount {
         self.0.account.owner()
     }
 
-    pub fn vote_state(&self) -> Result<&VoteState, &Error> {
-        // VoteState::deserialize deserializes a VoteStateVersions and then
-        // calls VoteStateVersions::convert_to_current.
-        self.0
-            .vote_state
-            .get_or_init(|| VoteState::deserialize(self.0.account.data()).map_err(Error::from))
-            .as_ref()
-    }
-
-    pub fn is_deserialized(&self) -> bool {
-        self.0.vote_state.get().is_some()
+    pub fn vote_state(&self) -> &VoteState {
+        &self.0.vote_state
     }
 
     /// VoteState.node_pubkey of this vote-account.
-    pub fn node_pubkey(&self) -> Option<&Pubkey> {
-        self.vote_state().ok().map(|s| &s.node_pubkey)
+    pub fn node_pubkey(&self) -> &Pubkey {
+        &self.0.vote_state.node_pubkey
+    }
+
+    #[cfg(feature = "dev-context-only-utils")]
+    pub fn new_random() -> VoteAccount {
```

Comment:
It is relevant tho - without it tests fails, since they're doing VoteAccount::try_from(empty_account).unwrap() which worked before (lazy parse) and doesn't anymore (try_from does the parsing and an empty account isn't a valid vote account)

Reply:
Ah got it, thanks for clarifying, never mind then